docs/digger/admin/using-http-proxy.asciidoc

```diff
@@ -2,25 +2,26 @@
 
 When you configure the Digger Installer to use proxy, it will automatically reconfigure the following components:
 
-
 * The Jenkins server
 * The Android SDK pod used to download Android SDKs
-* Container/node that will run the build jobs, by adding the appropriate environment variables like `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file - sample provided https://github.com/aerogear/digger-installer/blob/master/inventory-sample[here].
-* Ansible to install various tools used to provision the macOS servers(??? verify if this is correct or it should be "Ansible to install various macOS tools"
+* Container/node that will run the build jobs, by adding the appropriate environment variables like `HTTP_PROXY` and `HTTPS_PROXY` in the inventory file. 
+Sample provided in https://github.com/aerogear/digger-installer/blob/master/inventory-sample[AeroGear Digger GitHub repo].
+* Ansible to install various tools used to provision the macOS servers
```

Comment:
(??? verify if this is correct or it should be "Ansible to install various macOS tools")

Reply:
Yes agreed the wording is funny here. Ansible shouldn't be listed under the heading `the following components` See [digger-installer README](https://github.com/aerogear/digger-installer/blob/d47dfcf0c912beaf2f06655b12a27ce77528903e/provision-osx/README.md#configure-proxy) for more information on proxy configurtaion in macOS context

Reply:
@laurafitzgerald how/by which component is Ansible (re)configured?

Reply:
@pmacko1 can you ping me via message on this one. Not sure what the question is. Lets have a discussion about it.

Reply:
@laurafitzgerald will ping you here as well, just to have everything in one place

if/whether Installer can modify Ansible settings ("self" reconfiguration, bc the installer is being run by ansible, kind of a feedback loop?) ("it will automatically reconfigure the following components)

Reply:
@laurafitzgerald macOS servers require manual configuration/setup
q: What Ansible does regarding configuring macOS and proxy?
https://github.com/aerogear/digger-installer/tree/d47dfcf0c912beaf2f06655b12a27ce77528903e/provision-osx#configure-proxy
Does it have to be configured completely by hand?

Reply:
ping @laurafitzgerald @matskiv

Reply:
So why Ansible should not be listed here? @matskiv @laurafitzgerald

Reply:
Installer is not configuring Ansible in any way. It should say something like: 

> * macOS servers used to run iOS builds

Reply:
resolved